sdk-commons/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java

```diff
@@ -85,17 +87,11 @@ public void sendReport(Wizard wizard) throws MojoExecutionException {
      * @return
      */
     private boolean checkIfOneWeekFromLastReport(){
-        Date lastReport;
-        try {
-            lastReport = getLastReported();
-        } catch (ParseException e) {
-            lastReport = null;
-        }
-
-        Date currentDate = new Date();
+        LocalDate lastReport = getLastReported();
+        LocalDate currentDate = LocalDate.now();
 
         if (lastReport != null) {
-            return  (currentDate.getTime() - lastReport.getTime()) / (60 * 60 * 1000 * 24) % 60 > 7;
```

Comment:
Can we add some tests for things like this?

Reply:
@dkayiwa this method is private, Should I write tests for [this] (https://github.com/openmrs/openmrs-sdk/blob/806a9e6dd3920144edd78a712248ec9e63d7b25c/sdk-commons/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java#L70) or use reflection?

Reply:
Is the method that calls it public? If yes, does it have a test?

Reply:
This gets called by the following method.

https://github.com/openmrs/openmrs-sdk/blob/806a9e6dd3920144edd78a712248ec9e63d7b25c/sdk-commons/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java#L70

This method does not have any tests. It sends a report to Google Forms.

Reply:
Can you create some tests for it?